test/addons/cdi/fetch

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from drenv import cache
+
+os.chdir(os.path.dirname(__file__))
```

Comment:
You added a second block of code separated by blank line, so separating the chdir line with a blank line would be nice.

Same for the other new fetch hook.